lms/static/sass/elements/_course-card.scss

```diff
@@ -47,8 +47,26 @@
       margin-bottom: $baseline/2;
       text-transform: uppercase;
     }
-
-    .run-select-container {
+    .select-error{
+      color: palette(error, base);
+      margin-bottom: $baseline/4;
+      font-size: font-size(small);
+      visibility: hidden;
```

Comment:
Yes, we want to have the space for the error message on top of the select, and more importantly, if the error message shows, it does not push the select and the green button down. Therefore I used visibility.